src/components/atoms/Button/index.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Button = (props) => {
+  const { children, ...rest } = props;
+
+  return <button {...rest}>{children}</button>;
```

Comment:
```suggestion
  const { children,  } = props;

  return <button {...rest} />;
```

또잉 이 정도면 그냥 button element 아닌가요?ㅋㅋㅋ
props에 정의된 것들을 보니 button element의 attribute와 다른게 없어보이는데, 요 컴포넌트는 왜 필요할까요?!ㅎㅎ

Reply:
뜨악 ㅎㅎㅎ 😭
이 친구를 처음 만든 것은 `button 요소의 기본 스타일링을 없애고, 배경색 등 자주 쓰이는 공통 스타일을 부여`하려는 의도였습니다...!

그런데 button 요소의 기본 스타일링을 없애는 것은 global style로 적용해서 해결되고, 공통 스타일은 `ButtonRound`, `ButtonSquare`에서 미처 빼지 못해서 중복인 채로 남아있었네요... 그리고 아무일도 하지 않고 숨만 쉬는 컴포넌트가 생겨버렸,,,🥲

원래의 의도대로 `ButtonRound`, `ButtonSquare`에서 공통 스타일을 추출해서 Button에 입혀주었습니다. 감사합니다 :)
📌 관련 커밋: b452b15

\+ 혹시 `suggested change`로 올려주신 코드는 button 요소의 `children`을 코드 상에 명시하지 않고 그대로 `...rest`로 보내는 것을 추천해주신 것으로 이해하면 맞을까용...!

Reply:
넹~